src/main/java/com/shaft/dsl/gui/SearchField.java

```diff
@@ -0,0 +1,17 @@
+package com.shaft.dsl.gui;
+
+import org.openqa.selenium.By;
+
+public class SearchField extends TextBox {
+   public Button clearBtn;
+
```

Comment:
I don't think the search field is a standard DSL expression, can you tell me more about it or remove it?

Reply:
Search field is a textbox with close button that have an id
so it is a domain component not a text field 
also it is a part of another components coming in progress

Reply:
I think it's project specific though... didn't see it before

Reply:
it is very common in the portals with modern 
![image](https://user-images.githubusercontent.com/77445441/155281929-be320665-6be5-4cfc-ad2d-9ba39f119628.png)

Reply:
![image](https://user-images.githubusercontent.com/77445441/155282068-c4f1296e-f920-4d4f-a716-e370cab6453f.png)
it is W3C element

Reply:
I can remove it , but I will need it extensively in coming components; filters , searching and so

Reply:
to me, this still looks like a normal text input field (albeit a single line one) the only difference is that you can press enter to search, or click a button to search, or perform .submit...

seems to me that having a regular input field in SHAFT with the ability to type, submit, or perform a key press is sufficient for this need.

can we delay the discussion and remove this component for now so that I can generate the new build? I think you have a valuable point... I'm not getting it yet... and I want to push the new release for you... so let's exclude this file for now but keep the conversation going.

sounds good?

Reply:
fair enough

Reply:
done